src/transformers/integrations/awq.py

```diff
@@ -328,6 +335,8 @@ def _fuse_awq_attention_layers(model, module, modules_to_fuse, current_module_na
             previous_device,
             modules_to_fuse["max_seq_len"],
             use_alibi=modules_to_fuse["use_alibi"],
+            # The default value in autoawq is set to 10000.0
+            rope_theta=modules_to_fuse.get("rope_theta", 10000.0),
```

Comment:
This specifically addresses: https://github.com/casper-hansen/AutoAWQ/pull/251#issuecomment-1867842168

Reply:
Yes correct, let's keep that in mind, cc @casper-hansen for visibility